src/qt/rpcconsole.cpp

```diff
@@ -471,14 +471,28 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
 
     constexpr QChar nonbreaking_hyphen(8209);
     const std::vector<QString> CONNECTION_TYPE_DOC{
+        //: Explanatory text for an Inbound peer connection.
         tr("Inbound: initiated by peer"),
+        /*: Explanatory text for an Outbound peer connection which
```

Comment:
s/Inbound/inbound/ and s/Outbound/outbound/ in these descriptions when used as an adjective and not as a proper noun, which seems to be the case throughout.